### PR TITLE
Update badges in readme

### PR DESCRIPTION
**Pull request by xPaw**
New:

Old:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-[![#thelounge IRC channel on freenode](https://img.shields.io/badge/irc%20channel-%23thelounge%20on%20freenode-blue.svg)](https://avatar.playat.ch:1000/)
-[![npm version](https://img.shields.io/npm/v/thelounge.svg)](https://www.npmjs.org/package/thelounge)
-[![Travis CI Build Status](https://travis-ci.org/thelounge/lounge.svg?branch=master)](https://travis-ci.org/thelounge/lounge)
-[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/deymtp0lldq78s8t/branch/master?svg=true)](https://ci.appveyor.com/project/astorije/lounge/branch/master)
-[![Dependency Status](https://david-dm.org/thelounge/lounge.svg)](https://david-dm.org/thelounge/lounge)
-[![devDependency Status](https://david-dm.org/thelounge/lounge/dev-status.svg)](https://david-dm.org/thelounge/lounge?type=dev)
-
 # The Lounge
 
+[![](https://img.shields.io/badge/freenode-%23thelounge-BA68C8.svg)](https://avatar.playat.ch:1000/)
```
Comment (astorije) [inline README.md]: Don't remove alternative text please.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+[![](https://img.shields.io/npm/v/thelounge.svg)](https://www.npmjs.org/package/thelounge)
+[![](https://img.shields.io/travis/thelounge/lounge/master.svg?label=linux+build)](https://travis-ci.org/thelounge/lounge)
```
Comment (astorije) [inline README.md]: What about just "linux" and "windows"?
Anyhow, that will work only until we re-add OSX to Travis CI (if we ever do) :P
Alternative would be "travis ci" and "appveyor" though.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+[![](https://img.shields.io/appveyor/ci/astorije/lounge/master.svg?label=windows+build)](https://ci.appveyor.com/project/astorije/lounge/branch/master)
+[![](https://img.shields.io/david/thelounge/lounge.svg)](https://david-dm.org/thelounge/lounge)
+[![](https://img.shields.io/david/dev/thelounge/lounge.svg)](https://david-dm.org/thelounge/lounge?type=dev)
+
```
Comment (astorije) [inline README.md]: I'm not too convinced on using from Shields directly but not frankly a big deal 😅

File: README.md (hunk 2 above)
Comment (xPaw) [inline README.md]: We can change it back to CI names if we add OSX.

**Pull request merged**